src/PIL/_typing.py

```diff
@@ -15,4 +16,7 @@ def __class_getitem__(cls, item: Any) -> type[bool]:
                 return bool
 
 
+CoordList = Union[Sequence[float], Sequence[Sequence[float]]]
```

Comment:
Maybe just `Coords`? It's shorter, and a sequence can be a not-a-list such as a tuple:

```suggestion
Coords = Union[Sequence[float], Sequence[Sequence[float]]]
```

Reply:
Sure, I've made this change.